lib/Froxlor/Api/Commands/Certificates.php

```diff
@@ -334,7 +334,7 @@ private function addOrUpdateCertificate($domainid = 0, $ssl_cert_file = '', $ssl
 			} else {
 				\Froxlor\UI\Response::standard_error('sslcertificateinvalidcert', '', true);
 			}
-			$expirationdate = $cert_content['validTo_time_t'];
+			$expirationdate = empty($cert_content['validTo_time_t']) ? null : $date("Y-m-d H:i:s", $cert_content['validTo_time_t']);
```

Comment:
```diff
- $date(...
+ date(...
```

error in unittests: **Undefined variable: date**

Reply:
Fixed, I'm very sorry... it's better for me to go offline for today :)

Thanks!

Reply:
No need to be sorry, happens :)